foyer-memory/src/cache.rs

```diff
@@ -515,6 +521,19 @@ where
     }
 }
 
+impl<K, V> Cache<K, V>
+where
+    K: Key,
+    V: Value,
+{
+    /// Create a new in-memory cache builder.
+    ///
+    /// The default capacity is 1MiB, use [`CacheBuilder::with_capacity`] to set the capacity.
+    pub fn builder() -> CacheBuilder<K, V, RandomState> {
+        CacheBuilder::new(1024 * 1024)
```

Comment:
Since `capacity` is a really important argument. Is it okay to require the `capacity` here when creating the builder?

Reply:
That makes sense. Let me add it.